### PR TITLE
Replace 'minio' with a generic term in sample YAML files

### PR DESCRIPTION
**Pull request by RichardHoch**
OADP 1.4.1 (OK to merge and publish before GA), OCP 4.13+

Resolves https://issues.redhat.com/browse/OADP-1322 by replacing `minio` by a more generic term in 3 sample YAML files.

Previews:

- https://79342--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.html#oadp-secrets-for-different-credentials_installing-oadp-mcg [step 5]
- https://79342--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.html#oadp-installing-dpa-1-2-and-earlier_installing-oadp-mcg [step 3]
- https://79342--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.html#oadp-installing-dpa-1-3_installing-oadp-mcg [step 3]

Approved by OADP QA [PrasadJoshi12 below]

### CODE REVIEW

**Comment by RichardHoch** [comment]
@PrasadJoshi12 Please review. Thanks.

**Review by stillalearner — APPROVED** [review]
lgtm

**Comment by tareqalayan** [comment]
looks good to me

File: modules/oadp-installing-dpa-1-2-and-earlier.adoc
Comment (anarnold97) [inline modules/oadp-installing-dpa-1-2-and-earlier.adoc:265]:
```suggestion
<5> Specify the region, following the naming convention of the documentation of your object storage server.
```

File: modules/oadp-installing-dpa-1-3.adoc
Comment (anarnold97) [inline modules/oadp-installing-dpa-1-3.adoc:281]:
```suggestion
<8> Specify the region, following the naming convention of the documentation of your object storage server.
```

File: modules/oadp-secrets-for-different-credentials.adoc
Comment (anarnold97) [inline modules/oadp-secrets-for-different-credentials.adoc:196]:
```suggestion
<1> Specify the region, following the naming convention of the documentation of your object storage server.
```

**Review by weshayutin — APPROVED** [review]
/LGTM

**Review by PrasadJoshi12 — APPROVED** [review]

**Review by shubham-pampattiwar — APPROVED** [review]

**Review by skopacz1 — APPROVED** [review]
So this PR didn't get a peer review, however I just gave it one and it LGTM. I'll proceed with merge reviewing this but next time, please formally get a peer review by adding the peer review needed label before you request a merge review. Thanks!

**Pull request merged**
